versioned_docs/version-0.9.0/getting_started/02_breakdown.md

```diff
@@ -119,7 +119,7 @@ nargo prove p
 This command looks for proof inputs in the custom **OtherProver.toml** and generates proof `pp`:
 
 ```bash
-nargo prove -p OtherProver pp 
+nargo prove -p OtherProver pp
```

Comment:
This is the versioned docs, so it probably doesn't need to be changed here, but the argument to `nargo prove` was removed so any usages in recent docs need to be adjusted.